### PR TITLE
fix this issue: Multiple ApolloClients don't work #1023

### PR DESCRIPTION
**Pull request by kahirokunn**

### CODE REVIEW

**Comment by kahirokunn** [comment]
@liaoliaojun 谢谢发现问题，是我没考虑好几个问题。

File: packages/vue-apollo-composable/src/useApolloClient.ts [hunk 1]
```diff
@@ -23,31 +23,35 @@ function resolveDefaultClient<T>(providedApolloClients: ClientDict<T>, providedA
...
-  const providedApolloClient: ApolloClient<TCacheShape> = inject(DefaultApolloClient, null)
+  const providedApolloClients = inject<ClientDict<TCacheShape>>(ApolloClients, null)
+  const providedApolloClient = inject<ApolloClient<TCacheShape>>(DefaultApolloClient, null)
 
-  function resolveClient() {
+  function resolveClient(clientId?: ClientId) {
     if (clientId) {
-      resolveClientWithId(providedApolloClients, clientId)
+      assertProvidedApolloClients(providedApolloClients, clientId)
+      return resolveClientWithId(providedApolloClients, clientId)
```
Comment (kahirokunn) [inline packages/vue-apollo-composable/src/useApolloClient.ts:46]: before just exec resolveClientWithId. not return.
And then resolveClient required optional clientId params for follow UseApolloClientReturn interface.

Comment (Akryum) [reply packages/vue-apollo-composable/src/useApolloClient.ts:46]: https://github.com/vuejs/vue-apollo/commit/72d7402eb29893a7e63fc675a8057a8f5a4a825f

**Comment by Akryum** [comment]
Thanks! Was fixed on v4 branch!

**Pull request closed without merging**